roles/keystone/templates/etc/keystone/uwsgi/keystone-admin.ini

```diff
@@ -25,3 +25,4 @@ wsgi-file = /bin/keystone-wsgi-admin
 {% else %}
 wsgi-file = /usr/local/bin/keystone-wsgi-admin
 {% endif %}
+buffer-size = 114688
```

Comment:
We should parameterize this with role default value of 114688
variable structure should be keystone.uwsgi.buffer_size